modules/router/manifests/nginx.pp

```diff
@@ -125,6 +125,13 @@
     require => File['/usr/share/nginx/www'],
   }
 
+  # HTML verification for DWP YouTube channel
+  file { '/usr/share/nginx/www/google6db9c061ce178960.html':
+    ensure  => present,
+    source  => 'puppet:///modules/router/google6db9c061ce178960.html',
```

Comment:
I think the contents can just be an empty string. We could do `content => ''` so that we don't need a separate file.